src/src/pages/capabilities/SelectedCapabilityContext.js

```diff
@@ -90,6 +94,13 @@ function SelectedCapabilityProvider({ children }) {
     kafkaClusterTopicList();
   }, [clustersList]);
 
+  // load bypass rights:
+  const getBypassMembershipApplication = useCallback(async () => {
+    const result =
+      await selfServiceApiClient.CheckCanBypassMembershipApproval(details);
```

Comment:
This is not asynchronous, is it?
Did you want to call `BypassMembershipApproval`